git/foundations_git/git_basics.md

```diff
@@ -6,20 +6,20 @@ In this lesson, we'll cover common Git commands used to manage your projects and
 
 This section contains a general overview of topics that you will learn in this lesson.
 
- - How to create a repository on GitHub
- - How to get files to and from GitHub
- - How to take "snapshots" of your code
+- How to create a repository on GitHub
+- How to get files to and from GitHub
+- How to take "snapshots" of your code
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-#### Before you start!
+#### Before you Start!
```

Comment:
```suggestion
#### Before You Start!
```
I used the Title Case converter with Wikipedia selected, apparently the "You" needs to be capitalized.